docs/content/reference/static-configuration/file.yaml

```
@@ -122,7 +122,7 @@ providers:
     debugLogGeneratedTemplate: true
   kubernetesIngress:
     endpoint: foobar
-    token: foobar
+    token: foobar or foobar.txt
```

In this file we are only adding example values, and for a string type it is always foobar

```suggestion
    token: foobar
```